src/EnergyPlus/PurchasedAirManager.hh

```diff
@@ -274,8 +276,7 @@ namespace PurchasedAirManager {
               EMSValueMassFlowRate(0.0), EMSOverrideOAMdotOn(false), EMSValueOAMassFlowRate(0.0),
               EMSOverrideSupplyTempOn(false), EMSValueSupplyTemp(0.0), EMSOverrideSupplyHumRatOn(false),
               EMSValueSupplyHumRat(0.0), MinOAMassFlowRate(0.0),
-              OutdoorAirMassFlowRate(0.0), OutdoorAirVolFlowRateStdRho(0.0), SupplyAirMassFlowRate(0.0), SupplyAirVolFlowRateStdRho(0.0),
-              FinalMixedAirTemp(0.0), FinalMixedAirHumRat(0.0), HtRecSenOutput(0.0), HtRecLatOutput(0.0), OASenOutput(0.0), OALatOutput(0.0),
+              OutdoorAirMassFlowRate(0.0), OutdoorAirVolFlowRateStdRho(0.0), SupplyAirMassFlowRate(0.0), SupplyAirVolFlowRateStdRho(0.0), HtRecSenOutput(0.0), HtRecLatOutput(0.0), OASenOutput(0.0), OALatOutput(0.0),
```

Comment:
Not sure why this line didn't wrap earlier. It's much longer than all the rest. Was Clang format used?

Reply:
@rraustad I just checked this, it was disabled in my IDE. Shall I do one more commit after reformatting?

Reply:
@Myoldmopar is it worth another commit for Clang formatting the hh file? I don't see any other changes needed, but I haven't tested the example file yet. Unit tests run clean locally.